.env.example

```diff
@@ -15,7 +15,8 @@ FORWARD_PHPMYADMIN_PORT=64080
 
 # Feature Flags
 
-FEATURE_BEAT=true
+## example
+#FEATURE_BEAT=true
```

Comment:
`#FEATURE_EXAMPLE=true`

Reply:
Resolved in https://github.com/RetroAchievements/RAWeb/commit/e57d45732f1ccd3c1694bb9eb63b62d26222a0f6.